perfiles_usuario/tests.py

```diff
@@ -1,3 +1,145 @@
-# from django.test import TestCase
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
 
-# Create your tests here.
+from .utils import is_member, is_administrador, is_capturista, is_directivo, is_servicios_escolares
+from .utils import ADMINISTRADOR_GROUP, CAPTURISTA_GROUP, \
+                DIRECTIVO_GROUP, SERVICIOS_ESCOLARES_GROUP
+from .models import Capturista
+
+
+class PermisosTestCase(TestCase):
+
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+                                        username='some_user',
+                                        email='temporary@gmail.com',
+                                        password='some_pass')
+        self.directivo_group = Group.objects.get_or_create(name=DIRECTIVO_GROUP)[0]
+        self.administrador_group = Group.objects.get_or_create(name=ADMINISTRADOR_GROUP)[0]
+        self.servicios_group = Group.objects.get_or_create(name=SERVICIOS_ESCOLARES_GROUP)[0]
```

Comment:
I would highly recommend the use of a short summary on this method, because at the end of the day it was written by a human. 

Also I believe the attributes should be documented, as they would directly help a person understand the setup required for the tests, in case he breaks them in the future.